tests/trainer/test_dataloaders.py

```diff
@@ -782,6 +782,8 @@ def test_warning_with_few_workers_multi_loader(_, tmpdir, ckpt_path, stage):
         match=f'The dataloader, {stage} dataloader{" 0" if stage != "train" else ""}, does not have many workers'
     ):
         if stage == 'test':
+            if ckpt_path == 'specific':
+                trainer.fit(model, train_dataloader=train_multi_dl, val_dataloaders=val_multi_dl)
```

Comment:
This test kinda followed the behaviour of before, where it assumed passing the ckpt_path actually did anything. Now that we actually load the ckpt_path, we have to train the model first to know what's the best model to use